src/components/app/StrategiesList/index.tsx

```diff
@@ -166,6 +167,12 @@ const _StrategiesList = (props: StrategiesListProps) => {
                                                     <Hidden xsDown>
                                                         {strategy.name}
                                                     </Hidden>
+
+                                                    <Hidden xsDown>
```

Comment:
If it does not distort the view on mobile, we can remove the `Hidden` tag, so its available there also